src/components/Button/Button.tsx

```diff
@@ -32,6 +32,10 @@ const Button = ({
   right,
   bottom,
 }: ButtonProps) => {
+  // handle buttons still using `outline` variant
+  if (variant === 'outline') {
+    variant = 'primary'
+  }
   const props = {
```

Comment:
I was looking through citrus and since there's quite a few outline variants I added this check.